lib/utilities/lang.js

```diff
@@ -2,7 +2,9 @@
 // Thanks Yahoo!
 // http://yuilibrary.com/yui/docs/api/files/yui_js_yui-lang.js.html
 
-var TOSTRING     = Object.prototype.toString;
+/*eslint-disable */
```

Comment:
Some other projects do this by having a "src" directory that is like our "lib" directory, and using "lib" for misc. third party utilities/libraries that aren't on npm. We could try something like that.

ESLint also lets us specify specific files which should not be considered (iirc it's like .eslintignore or something to that effect).